tools/integration_tests/mounting/gcsfuse_test.go

```diff
@@ -551,3 +558,154 @@ func (t *GcsfuseTest) HelpFlags() {
 		ExpectEq(nil, err, "case %d\nOutput:\n%s", i, output)
 	}
 }
+
+const TEST_RELATIVE_FILE_NAME = "test.txt"
+const TEST_HOME_RELATIVE_FILE_NAME = "test_home.json"
+
+func (t *GcsfuseTest) createTestFilesForRelativePathTesting() (
+	curDirTestFile string, homeDirTestFile string) {
+
+	curDirTestFile = filepath.Join(t.dir, TEST_RELATIVE_FILE_NAME)
+	_, err := os.Create(curDirTestFile)
+	AssertEq(nil, err)
+
+	homeDir, err := os.UserHomeDir()
+	AssertEq(nil, err)
+
+	homeDirTestFile = filepath.Join(homeDir, TEST_HOME_RELATIVE_FILE_NAME)
+	_, err = os.Create(homeDirTestFile)
+	AssertEq(nil, err)
+
+	return
+}
+
+func (t *GcsfuseTest) RelativeLogFilePath() {
```

Comment:
rename this method to logfilePath and add absolute path test also.

same for key file